library/src/commonMain/kotlin/com/lagradost/cloudstream3/extractors/Vidplay.kt

```diff
@@ -1,13 +1,16 @@
 package com.lagradost.cloudstream3.extractors
 
+import android.util.Base64
```

Comment:
This will fail, use the built in base64 helper as that works without android.

Reply:
inbuilt Base64Encode is working well but Base64Decode is having some issue so I switched to Kotlin.